app/controllers/data_access_controller.rb

```diff
@@ -50,10 +50,22 @@ def retrieve
     retrieval.user = user
     retrieval.project = project
     retrieval.save!
-
-    new_job = Retrieval.delay.process(retrieval.id, token, cmr_env, edsc_path(request.base_url + '/'), session[:access_token])
-    Rails.logger.info("A new delayed job with ID " + new_job.id.to_s + " has been created: " + params[:project]) 
-
+    
+    queue = "Default"
+    daacs = ENV["priority_daacs"] ? ENV["priority_daacs"].split(",") : []
```

Comment:
Good catch on this, I was going to comment on it. That being said I think a better way to handle this would be to make it a config in the application, its already hardcoded in the Procfile so its not going to provide us any flexibility by being an ENV var. Additionally we'll have to set this value in all of our local environments, on Travis, and on Bamboo, just a lot of overhead for something that isn't going to change often and isn't a secret key.

Reply:
@abbottry - considering the names are already in the Procfile, I *could* just include them in this code as well - just set daacs = ['NSIDC', 'LPDAAC'].  Other daacs could be added down the line as needed.  

What do you think?

Reply:
Sure, thats fine -- if it gets more complicated in the future we can pull them out.